src/components/GisidaLite/index.tsx

```diff
@@ -134,4 +125,19 @@ const GisidaLite = (props: GisidaLiteProps) => {
 
 GisidaLite.defaultProps = gisidaLiteDefaultProps;
 
-export { GisidaLite };
+/**
+ * Custom equality method for React.memo
+ * @param prevProps
+ * @param nextProps
+ */
+export const arePropsEqual = (prevProps: GisidaLiteProps, nextProps: GisidaLiteProps) => {
+  if (!isEqual(prevProps.layers, nextProps.layers)) {
+    return false;
+  }
+
+  return true;
+};
```

Comment:
can we just `return isEqual(prevProps.layers, nextProps.layers)` here.

Reply:
@ciremusyoka addressed